shuup/admin/templates/shuup/admin/base.jinja

```diff
@@ -19,6 +19,78 @@
+                        --primary-dark: {{ colorscale(admin_theme.primary_color, 0.88) }};
+                    {% endif %}
+                    {% if admin_theme.secondary_color %}
+                        --secondary: {{ admin_theme.secondary_color }};
+                        --secondary-light: {{ admin_theme.secondary_color }}70;
+                        --secondary-very-light: {{ admin_theme.secondary_color }}20;
+                        --secondary-lighter: {{ admin_theme.secondary_color }}50;
+                        --secondary-lightest: {{ admin_theme.secondary_color }}30;
+                        --secondary-opaque: {{ admin_theme.secondary_color }}10;
+                        --secondary-dark: {{ colorscale(admin_theme.secondary_color, 0.88) }};
+                    {% endif %}
+                    {% if admin_theme.success_color %}
+                        --success: {{ admin_theme.success_color }};
+                        --success-light: {{ admin_theme.success_color }}70;
+                        --success-lighter: {{ admin_theme.success_color }}50;
+                        --success-lightest: {{ admin_theme.success_color }}30;
+                        --success-very-light: {{ admin_theme.success_color }}20;
+                        --success-opaque: {{ admin_theme.success_color }}10;
+                        --success-dark: {{ colorscale(admin_theme.success_color, 0.88) }};
+                    {% endif %}
+                    {% if admin_theme.danger_color %}
+                        --danger: {{ admin_theme.danger_color }};
+                        --danger-light: {{ admin_theme.danger_color }}70;
+                        --danger-lighter: {{ admin_theme.danger_color }}50;
+                        --danger-lightest: {{ admin_theme.danger_color }}30;
+                        --danger-very-light: {{ admin_theme.danger_color }}20;
+                        --danger-opaque: {{ admin_theme.danger_color }}10;
+                        --danger-dark: {{ colorscale(admin_theme.danger_color, 0.88) }};
+                        --danger-darker: {{ colorscale(admin_theme.danger_color, 0.80) }};
+                    {% endif %}
+                    {% if admin_theme.danger_color %}
+                        --text-color: {{ admin_theme.text_color }};
+                        --text-color-light: {{ admin_theme.text_color }}70;
+                        --text-color-lighter: {{ admin_theme.text_color }}50;
+                        --text-color-lightest: {{ admin_theme.text_color }}30;
+                        --text-color-very-light: {{ admin_theme.text_color }}20;
+                        --text-color-opaque: {{ admin_theme.text_color }}10;
+                        --text-color-dark: {{ colorscale(admin_theme.text_color, 0.88) }};
+                    {% endif %}
+                }
+                h1, h2, h3, h4{
+                    {% set header_font = get_font(request, admin_theme.admin_header_font_id) %}
```

Comment:
you don't need this template tag, you can pretty much access it directly: `admin_theme.admin_header_font`

Reply:
✅